packages/strapi-plugin-users-permissions/admin/src/components/Policies/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { useMemo } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { get, isEmpty, takeRight, toLower, without } from 'lodash';
+import { InputsIndex as Input } from 'strapi-helper-plugin';
+
+import { useUsersPermissions } from '../../contexts/UsersPermissionsContext';
+import BoundRoute from '../BoundRoute';
+import { Header, Wrapper } from './Components';
+
+const Policies = () => {
+  const { selectedAction, routes, permissions, policies, onSelectedPolicy } = useUsersPermissions();
+  const baseTitle = 'users-permissions.Policies.header';
+  const title = !selectedAction ? 'hint' : 'title';
+  const path = without(selectedAction.split('.'), 'controllers');
+  const controllerRoutes = get(routes, path[0]);
+  const displayedRoutes = isEmpty(controllerRoutes)
+    ? []
+    : controllerRoutes.filter(o => toLower(o.handler) === toLower(takeRight(path, 2).join('.')));
```

Comment:
Maybe you could change the `selectedAction` type so it is an object: 

```
const selectedAction = { 
  pluginName,  // null if it is application, name of the plugin otherwise
  handler,
  method,
  path, 
  config,
}
```

By doing so you won't be needing the `displayedRoutes` array as one route is always bound to one controller you should not be needing an array here.